### PR TITLE
Update security-findings-faq.md

### PR DESCRIPTION
**Pull request by MxClaudineMaas**
https://mendixsupport.zendesk.com/agent/tickets/178085

### CODE REVIEW

**Comment by MxClaudineMaas** [comment]
Oh the latest should be 8.6.2 instead of 8.6.1 but i dont know how to change it

**Comment by Adam-Dupaski** [comment]
@MarkvanMents is this one for you?

File: content/en/docs/developerportal/support/security-findings-faq.md [hunk 1]
```diff
@@ -194,6 +194,12 @@ For example, the old version of the [LDAP Synchronization module](https://market
...
-### 8.6 Dojo library - deepCopy Vulnerability - CVE-2020-5258
+### 8.6 Dojo library 
+
+#### 8.6.1 deepCopy Vulnerability - CVE-2020-5258
 
 The Mendix Client is bundled with the full dojo library. However, not all functionality of the Dojo library is used. The vulnerability is in the `deepCopy` util method of dojo; this method is not used in the Mendix Client. This vulnerability cannot be exploited in the client.
+
+#### 8.6.2 Prototype Pollution Vulnerability - CVE-2021-23450
+
+Mendix 9.12.4 uses Dojo 1.16.4. The `setObject` function is used in one place in our code, but this is in a deprecated function. This function is no longer used internally. Therefore, there is no opportunity for the `setObject` function to be exploited.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/support/security-findings-faq.md]: @MxClaudineMaas 
Is this only relevant for 9.12.4 or are there other versions of Mendix which would also give this scan result?

Comment (MarkvanMents) [reply content/en/docs/developerportal/support/security-findings-faq.md]: 9.11 and above...

**Comment by MarkvanMents** [comment]
Agreed with Claudine over Slack.

**Review by MarkvanMents — APPROVED** [review]

**Pull request merged**